test/test_initial_design/test_factorical_design.py

```diff
@@ -0,0 +1,52 @@
+import unittest
+import unittest.mock
+
+import numpy as np
+from ConfigSpace import ConfigurationSpace, UniformFloatHyperparameter,\
+    Constant, CategoricalHyperparameter, OrdinalHyperparameter
+
+from smac.initial_design.factorial_design import FactorialInitialDesign
+
+
+class TestFactorial(unittest.TestCase):
+    def test_factorial(self):
```

Comment:
maybe 1-2 sentences what you are testing here

Reply:
I am not testing how it works per se but I am ensuring that all lines work without breaking